resources/win-chocolatey/tools/chocolateyinstall.ps1

```diff
@@ -20,9 +20,17 @@ if (Test-Path "${env:ProgramFiles(x86)}\Yarn\package.json") {
 } else {
   $path = "$env:ProgramFiles\Yarn\package.json"
 }
+
 $script = @"
   (Get-Content -Path '$path') ``
     -replace 'installationMethod":.+', 'installationMethod": "choco",' ``
     | Set-Content '$path'
 "@
+
 Start-ChocolateyProcessAsAdmin -Statements $script
+
+if (-Not (Get-Command "node" -errorAction SilentlyContinue)) {
+  Write-Host "Yarn requires NodeJS to be installed. To install, use either of the commands below:"
+  Write-Host "choco install nodejs"
+  Write-Host "choco install nodejs-lts"
```

Comment:
I would suggest to add this information also to the nuspec file as a note (like e.g. done [here](https://github.com/bbtsoftware/chocolatey-packages/blob/master/automatic/bower/bower.nuspec#L21-L24))